lib/beacon_api/utils.ex

```diff
@@ -10,4 +10,25 @@ defmodule BeaconApi.Utils do
   def is_bytes32?(value) when is_binary(value) do
     String.starts_with?(value, "0x") and byte_size(value) == 66
   end
+
+  @spec parse_id(binary) :: atom | binary
+  def parse_id("genesis"), do: :genesis
+  def parse_id("justified"), do: :justified
+  def parse_id("finalized"), do: :finalized
+  def parse_id("head"), do: :head
+
+  def parse_id("0x" <> hex_root) do
+    if byte_size(hex_root) == 66 do
```

Comment:
Shouldn't this be 64, since we already removed the leading "0x" ?

Also, I'm nitpicking, but the if could be moved to the clause guard:

```elixir
def parse_id("0x" <> hex) when byte_size(hex) == 64 do ...
def parse_id("0x" <> _hex), do: :invalid_id
```